spec/react_on_rails/utils_spec.rb

```
@@ -6,14 +6,139 @@
   RSpec.describe Utils do
+    # Github Actions already run rspec tests two times, once with shakapacker and once with webpacker.
+    # If rspec tests are run locally, we want to test both packers.
+    # If rspec tests are run in CI, we want to test the packer specified in the CI_PACKER_VERSION environment variable.
+    # Check script/convert and .github/workflows/rspec-package-specs.yml for more details.
+    PACKERS_TO_TEST = if ENV["CI_PACKER_VERSION"] == "old"
+                        ["webpacker"]
+                      elsif ENV["CI_PACKER_VERSION"] == "new"
+                        ["shakapacker"]
+                      else
+                        ["shakapacker", "webpacker"]
+                      end
```

rubocop (1.69.1)

[warning] Do not define constants this way within a block.

(Lint/ConstantDefinitionInBlock)